xclim/indicators/atmos/_precip.py

```diff
@@ -291,17 +294,58 @@ class HrPrecip(Hourly):
 )
 
 
-fire_weather_indexes = FireWeather(
-    identifier="fwi",
+def fire_weather_indexes(*args, **kwargs):
```

Comment:
I don't think this will work! `fire_weather_indexes` is supposed to be an Indicator object and not a pure callable. Rather, I would use the deprecation indice you created!

Reply:
I also had an idea to add a `_deprecated` field to the `Indicator` class that triggers a formatted `DeprecationWarning` if the field is non-empty. This way, we have the possibility of exclusively deprecating an Indicator even if the indice it's based on remains valid.

It would need access to the `identifier` so it could be right at the top o `__new__`. What do you think?

Reply:
This doesn't seem to me like something will do often, so I think the current solution is good enough.
Also, I think we'd want the `DeprecationWarning` to fire only in `__call__`, and not at import time. In which case, your suggestion could make the code a bit more elegant, but it wouldn't change the result on the user-side.